themes/nord-color-theme.json

```
@@ -1215,6 +1215,45 @@
         "foreground": "#81A1C1"
       }
     },
+    {
+      "name": "[Rust] Entity types",
+      "scope": [
+        "source.rust entity.name.type"
+      ],
+      "settings": {
+        "foreground": "#8FBCBB"
+      }
+    },
+    {
+      "name": "[Rust] Macro",
+      "scope": [
+        "meta.macro.rust"
+      ],
+      "settings": {
+        "fontStyle": "bold",
+        "foreground": "#88C0D0"
+      }
+    },
+    {
+      "name": "[Rust] Attributes",
+      "scope": [
+        "meta.attribute.rust",
+        "meta.attribute.rust punctuation",
+        "meta.attribute.rust keyword.operator"
+      ],
```

```suggestion
      "scope": [
        "source.rust meta.attribute",
        "source.rust meta.attribute punctuation",
        "source.rust meta.attribute keyword.operator"
      ],
```

Using `source.rust` makes it more explicit and reduces the need to use the `rust` selector more than once.